src/TestItemRunner.jl

```diff
@@ -59,7 +59,29 @@ end
     @test TestItemRunner.compute_line_column(content, 11) == (line=3, column=3)
 end
 
-function run_testitem(filepath, use_default_usings, package_name, original_code, line, column)
+struct TestSetupModuleSet
+    modules::Set{Symbol}
+    lock::ReentrantLock
+end
+
+const TEST_SETUP_MODULE_SET = TestSetupModuleSet(Set{Symbol}(), ReentrantLock())
```

Comment:
Hmmmm, while I agree with the sentiment, I'm not sure it makes a difference in practice, since we're eval-ing the modules into `Main.TestSetups` anyway? or maybe we create a new run_tests module that we eval test setups into each run. I'll try that.